cf-key/cf-key-functions.c

```diff
@@ -229,9 +229,7 @@ static bool KeepKeyPromisesRSA(RSA *pair, const char *public_key_file, const cha
 
     Log(LOG_LEVEL_VERBOSE, "Writing private key to '%s'", private_key_file);
 
-    const EVP_CIPHER *cipher = EVP_des_ede3_cbc();
-    int res = PEM_write_RSAPrivateKey(fp, pair, cipher, (void *)PRIVKEY_PASSPHRASE,
-                                 PRIVKEY_PASSPHRASE_LEN, NULL, NULL);
+    int res = PEM_write_RSAPrivateKey(fp, pair, NULL, NULL, 0, NULL, NULL);
```

Comment:
This is hardly the only place that needs to be changed. Maybe you forgot to add some changes to the commit?

Reply:
Actually, this is the only place where we generate a key. As noted in the commit message, the read functions remain untouched as they will be able to read both old encrypted keys as well as new unencrypted keys.

Reply:
I added some comments in other places to clarify.